src/translator.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+responsible for translating the article title into 'owo speak'
+*/
+
+class translator
+{
+    /*
+    responsible for adding stutter to the article title.
+    This stutter has a 30% chance of occuring on the first word of the title
+    and voids the chance if the first word of the title starts with a illigal character ie " ' " or ' " ' .
+    */
+    public function stutter($str)
+    {
+        if (mt_rand(1, 100) <= 30) {
+            $split = explode(" ", $str);
+            $firstLetter = substr($split[0], 0, 1);
+
+            if (!preg_match("/[a-zA-Z]/", $firstLetter)) {
+                return $str;
+            }
+
+            $generate = join(" ", $split);
+            $str = "$firstLetter-$firstLetter-$generate";
+        }
+        return $str;
+    }
+
+    /*
+    responsible for translating the article title into 'uwu speak'.
+    This is simpely done by replacing all occurences of 'r' and 'l' with a 'w' respectively.
+    */
+    public function translateToUWU($str)
+    {
+        $pattern = '/[lr]/i';
+        $str = preg_replace($pattern, 'w', $str);
+
+        return $str;
```

Comment:
The `$str` variable can be omitted here with directly returning `preg_replace($pattern, 'w', $str);`

Reply:
Commit addressing capilisation of this function ([a758cfc](https://github.com/Electrenator/Discord-uWuFication/commit/a758cfcdb93a53dcaf3533ab4739391e2c5c8e22)) also fully resolves this when merged.